ads/_config.js

```diff
@@ -207,6 +207,17 @@ export const adConfig = {
     prefetch: 'https://s.ato.mx/p.js',
   },
 
+  bannerflow: {
+    prefetch: [
+      'https://scripts.host.bannerflow.com/1.0.4/bf.min.js',
+      'https://scripts.host.bannerflow.com/1.1.4/render.min.js',
+    ],
+    preconnect: [
+      'https://resources.host.bannerflow.com',
+      'https://banners.host.bannerflow',
+    ],
+  },
```

Comment:
We have a renderStart [API](https://github.com/ampproject/amphtml/blob/master/ads/README.md#available-apis) that help to improve user experiences. We highly recommend implementing it. Please let me know if there's any question.

Reply:
We will implement this in our ad scripts in the near future.